### DISCUSSION TITLE
Unable to post kata

### DISCUSSION THREAD

**Discussion by Extend777** [codewars/codewars.com#2699] [General]
I have the same problem, maybe someone else has come across it. I came up with my own kata, made a description, wrote code, tests, and eventually noticed that I didn’t have a “publish” button. I seem to have enough honor (1800). I can't figure out what's wrong.

**Comment by hobovsky** [comment]
What is your Codewars username?

**Reply by Extend777** [reply]
It's link on my profile: https://www.codewars.com/users/Extend

**Reply by Extend777** [reply]
Problem is solved, it's ok)
